data/json/recipes/recipe_deconstruction.json

```diff
@@ -2201,7 +2201,7 @@
   "reversible": true,
   "autolearn": false,
   "qualities" : [ {"id":"SAW_M","level":1,"amount":1} ],
-  "tools": [ [ [ "welder", 500 ], [ "toolset", 750 ] ] ],
+  "tools": [ [ [ "welder", 500 ], [ "toolset", 750 ], [ "welder_crude", 750 ] ] ],
```

Comment:
It's a deconstruction recipe.
Welder data is more of a convention here - it wouldn't be used anyway, as welder tool requirement is changed into metal sawing requirement on deconstruction.